source/includes/connect/stable-api.php

```diff
@@ -0,0 +1,13 @@
+// start-specify-v1
```

Comment:
```suggestion
<?php

// start-specify-v1
```

Adding back the opening PHP tag so the file itself is valid PHP code. The only problem with the previous state of this file was the duplicate `<?php` tag.